code/lib/preview-api/src/modules/preview-web/render/StoryRender.test.ts

```diff
@@ -103,48 +103,4 @@ describe('StoryRender', () => {
     await render.renderToElement({} as any);
     expect(story.playFunction).not.toHaveBeenCalled();
   });
-
-  it('passes the initialArgs to loaders and render function if forceInitialArgs is true', async () => {
```

Comment:
This functionality moved to the store so not really much point unit testing it here. We have a unit test in the story and an integration test in `PreviewWeb.test.ts`